types/node/timers.d.ts

```diff
@@ -62,6 +62,7 @@ declare module 'timers' {
                 [Symbol.toPrimitive](): number;
             }
         }
+        /** Sets a timer which executes a function or specified piece of code once the timer expires. **/
         function setTimeout<TArgs extends any[]>(callback: (...args: TArgs) => void, ms?: number, ...args: TArgs): NodeJS.Timeout;
```

Comment:
We should probably use the docs straight from the nodejs docs.

Reply:
ping @orta